.travis.yml

```diff
@@ -10,5 +10,5 @@ install:
 - pip install -r requirements.txt
 
 script:
-- pycodestyle --max-line-length=99 app/*.py app/*/*.py
+- pycodestyle --max-line-length=99 -r *.py app/*.py
```

Comment:
Shouldn't Travis be running `pytest` here?

Reply:
You should also configure coveralls, to measure test coverage, and prevent merging PRs that lead to decrease in coverage. See example here: https://github.com/boutiques/boutiques/blob/master/.travis.yml#L53-57

Reply:
Good suggestion, doing so now.

Reply:
Added as you can see below.